src/app/InteractionModelEngine.h

```diff
@@ -54,7 +54,7 @@ namespace app {
 
 constexpr size_t kMaxSecureSduLengthBytes = 1024;
 /* TODO: https://github.com/project-chip/connectedhomeip/issues/7489 */
-constexpr uint32_t kImMessageTimeoutMsec = 12000;
+constexpr uint32_t kImMessageTimeoutMsec = 1000;
```

Comment:
is it too aggressive to set it as 1 seconds?

Reply:
see this commit https://github.com/project-chip/connectedhomeip/commit/9d74bd044e0ea84388f48541c437ba562c7b2238

Reply:
kImMessageTimeoutMsec needs to be less than gMessageIntervalMsec to make sense, otherwise it is always get overwritten and never take effect. 

Maybe we should allow timeout value to be configurable as issue::7489 suggested?

Reply:
yep, please make it configurable, thanks